cohort_4/playbook/ACTIVITIES.md

```diff
@@ -551,6 +551,16 @@ then).
 
 # Workshops
 
+## General Guidelines
+
+### Breaks
+
+If hackers don't take a break, they will likely get very antsy. You can either 
+
+- remind them to take breaks every hour / hour and a half
```

Comment:
I'm confused. If we don't want to baby the hackers, why are we suggesting to remind them to take breaks every hour or so?